lib/common/s3.sh

```diff
@@ -89,3 +89,53 @@ s3_ls() {
     $S3CMD ls $S3_BUCKET/$path/ | tr -s " " | cut -d' ' -f4 | sed -e "s#^$S3_BUCKET/$path/##"
 }
 export -f s3_ls
+
+# list recursively a folder on s3 and returns list of fullpath of file objects
+# $1 - path of folder on s3
+s3_ls_recur() {
+    local folder=${1%/}; shift # remove trailing slash
```

Comment:
Should also check that `$folder` is not empty! I tried running this with no arguments and got strange behaviour (it listed a bunch of SOOP-BA paths in `/mnt/imos-data/`.